src/Talis/Persona/Client/Base.php

```diff
@@ -55,40 +57,52 @@
     private $defaultTtl;
 
     /**
+     * @var string
+     */
+    private $appUserAgent;
+
+    /**
+     * @var string
+     */
+    private $phpVersion;
+
+    /**
      * Constructor
      *
      * @param array $config An array of options with the following keys: <pre>
      *      persona_host: (string) the persona host you'll be making requests to (e.g. 'http://localhost')
      *      persona_oauth_route: (string) the token api route to query ( e.g: '/oauth/tokens')
+     *      userAgent: Consuming application user agent string @since 2.0.0
+     *            examples: rl/5.2, rl, rl/5, rl/5.2 (php/5.3; linux/2.5)
```

Comment:
Do we need to know the release hash of TARL too? Not objecting, but do we just want to know what the consuming app is? If so, the release id's are usually bambooId-hash, so latest one was `1723-9095ba4`